index.js

```diff
@@ -18,20 +16,26 @@ app.listen(PORT, function(req, res){
 	console.log(`Server is running @ ${PORT}`);
 })
 
-
-
 app.get('/getWithYear', malaria);
 
-
 function malaria(req, res){
-	request(url, function(err, response, body){
+		var options ={
+			query: req.query.query,
+			start: req.query.start,
+			end: req.query.end,
+			sort: 'CITED desc',
+			pageSize:1,
+			synonym: true,
+			format: 'json'
+		}
+		request({url:reqUrl, qs:options }, function(err, response, body){
 		if(!err && response.statusCode == 200){
 			var data = JSON.parse(body);
-			var malaria= []
-			data.resultList.result.forEach(function(Result) {
-  				malaria.push({Title: Result.title, Author: Result.authorString, Year: Result.pubYear, Count: Result.citedByCount});  				
+				data.resultList.result.forEach(function(result) {
+  				var responseData = {Title: result.title, Author: result.authorString, Year: result.pubYear, Count: result.citedByCount};
+  				res.write(JSON.stringify(responseData));  				
```

Comment:
@Firnaa5 Don't stringify. and Why res.write? after the forEach is done. You can respond like `res.send` or `res.json`